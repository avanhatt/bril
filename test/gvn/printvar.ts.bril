test/ts/printvar.ts.bril