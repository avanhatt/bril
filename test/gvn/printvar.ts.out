void main() {
b1:
  v01: int = const 5;
  x1: int = id v01;
  print x1;
  ret ;
}
